src/foraging_gui/Foraging.py

```diff
@@ -3872,6 +3866,11 @@ def _Start(self):
         else:
             GeneratedTrials=self.GeneratedTrials
 
+            if self.Settings['AutomaticUpload']:
```

Comment:
I think the line number I told you got messed up because you removed lines elsewhere in the file. We want this block that generates the upload manifest to happen inside the `if self.StartANewSession==1 and self.ANewTrial==1` block. I would put it at the end of that block, just before the `else`